### PR TITLE
Add marathon to minimesosFile

### PR DESCRIPTION
**Pull request by pschorf**
Minimesos HEAD expects a marathon to be running and gets an NPE otherwise:
https://github.com/ContainerSolutions/minimesos/blob/master/minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java#L109

### CODE REVIEW

File: scheduler/minimesosFile [hunk 1]
```diff
@@ -72,6 +72,16 @@ minimesos {
         loggingLevel = "# INHERIT FROM CLUSTER"
     }
 
+    marathon {
+        imageName = "mesosphere/marathon"
+        imageTag = "v1.3.5"
+
+        // Add 'app { marathonJson = "<path or URL to JSON file>" }' for every task you want to execute
+        app {
+            marathonJson = "https://raw.githubusercontent.com/ContainerSolutions/minimesos/master/opt/apps/weave-scope.json"
```
Comment (dposada) [inline scheduler/minimesosFile]: Can we avoid starting up weave-scope?

File: scheduler/minimesosFile [hunk 2]
```diff
@@ -72,6 +72,16 @@ minimesos {
         loggingLevel = "# INHERIT FROM CLUSTER"
     }
 
+    marathon {
```
Comment (dposada) [inline scheduler/minimesosFile]: Can we add a comment about why this is needed even though we don't use it?

File: scheduler/minimesosFile (hunk 1 above)
Comment (pschorf) [inline scheduler/minimesosFile]: Done

File: scheduler/minimesosFile (hunk 2 above)
Comment (pschorf) [inline scheduler/minimesosFile]: Done

**Pull request merged**